src/arrayOfPeople.js

```diff
@@ -39,6 +39,15 @@
  */
 function arrayOfPeople(people) {
   // write code here
+  const room = [];
+  for (let i = 0; i < people.length; i++) {
+    if (room.includes(people[i])) {
```

Comment:
better to use `indexOf`

you can create variable for the result of `indexOf` and use it in `if` and `splice`